source/structured-streaming.txt

```diff
@@ -160,27 +213,83 @@ more about continuous processing, see the `Spark documentation <https://spark.ap
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To use continuous processing with the MongoDB Spark Connector, 
+         call the ``trigger()`` method on the ``writeStream`` property 
+         of the streaming Dataset or streaming DataFrame that you 
+         create from your MongoDB read stream. Pass a setting returned 
+         by ``Trigger.Continuous()`` to the the ``trigger()`` method.
+         
+         .. include:: /includes/fact-read-from-change-stream
+         
+         Specify read stream configuration settings on 
+         `DataStreamReader <https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/DataStreamReader.html>`__. You must specify the following 
+         configuration settings to read from MongoDB:
+         
+         .. list-table::
+            :header-rows: 1
+            :stub-columns: 1
+            :widths: 10 40
+         
+            * - Setting
+              - Description
+         
+            * - ``readStream.format()``
+              - The format to use for read stream data. Use ``mongodb``.
+         
+            * - ``writeStream.trigger()``
```

Comment:
Issue: is this a required setting for read stream configurations? According to the sentence:
"You must specify the following configuration settings to read from MongoDB:"
it seems required, but references a writeStream, not a readStream.